### DISCUSSION TITLE
Integrated tablet driver does not support my 32:9 correctly

### DISCUSSION THREAD

**Discussion by Raidoki** [ppy/osu#15593] [Q&A]
Hey,

im playing osu!lazer on a 32:9 5120x1440 screen and i noticed i cant use the integrated tablet driver correctly since the aspect ratio caps out on a too low value.

For example on stable osu! i use an area of 150x42 which is equal on a 16:9 of 75x42 but the integrated osu!lazer tablet driver caps out on 152. My tablet is an Wacom 480. If you need any more data let me know :)

edit: it works correctly but the issue is more of the small play area, if i want to use lets say an width of 160 which would equal 80 on a 16:9 it wont be possible since it caps out on 152

**Comment by peppy** [comment]
Please take a screenshot showing what you are attempting to achieve.

**Reply by Raidoki** [reply]
Thanks for the reply :) Ive attached the pictures with a small discription below, so the issue is with lazer im limited to half of the area because i have double the aspect ratio but the play area stays the same.


<img width="583" alt="2021-11-12 14_40_00-allgemein - Discord" src="https://user-images.githubusercontent.com/94165447/141477116-f1a1cc6c-df19-45c4-b01f-3da1f54ffc3d.png">
<img width="638" alt="2021-11-12 14_41_55-osu!" src="https://user-images.githubusercontent.com/94165447/141477135-fbc97c08-3a94-4e4a-9c39-94324c46082a.png">

**Reply by gagahpangeran** [reply]
Alternatively, just disable in-game tablet driver, then it will use your OS/system tablet driver.

**Reply by Raidoki** [reply]
> Alternatively, just disable in-game tablet driver, then it will use your OS/system tablet driver.

Ive done that now thanks! but it still would be cool to use the integrated driver :)

**Reply by peppy** [reply]
So to confirm you are trying to map your tablet in a way that some pieces of the screen are not reachable using the tablet? Do you use a mouse to navigate menus?

Would a better path forward potentially involve setting the in-game mapping to only map the playfield rather than the full screen bounds?

**Reply by Raidoki** [reply]
> So to confirm you are trying to map your tablet in a way that some pieces of the screen are not reachable using the tablet? Do you use a mouse to navigate menus?
> 
> Would a better path forward potentially involve setting the in-game mapping to only map the playfield rather than the full screen bounds?

yes, having an option to only map the playfield would be cool :)

**Comment by gagahpangeran** [comment]
> the integrated osu!lazer tablet driver caps out on 152

AFAIK, the 152 capped number that you refer is the physical width of the tablet. It is from open tablet driver, see https://github.com/OpenTabletDriver/OpenTabletDriver/blob/a36f4eabf87732a017216fc79f6741c216a97097/OpenTabletDriver.Configurations/Configurations/Wacom/CTH-480.json#L5
